fortinet-fortigate/package.yaml

```diff
@@ -21,8 +21,10 @@ pipelines:
     name: Onboard FortiGate logs via UDP
     description: A pipeline for ingesting and processing FortiGate logs received via UDP.
     definition: |
-      from "{{ inputs.syslog-url }}" read syslog
-      | parse content kv
-      | yield content
-      | set #schema = "fortinet.fortigate"
-      | publish "fortinet"
+      // tql2
+      load_http "{{ inputs.syslog-url }}"
+      read_syslog
```

Comment:
I think this works with `from`.

Reply:
I don't think `from` deduces `syslog`, does it?

Reply:
Now, but this:

```
from ... {
  read_syslog
}
```

I think we always want to use `from` and `to` if we can, only go back to the explicit way if we must.